src/util/email.js

```diff
@@ -121,6 +127,10 @@ module.exports = function(formio) {
 
       // Get the parameters for the email.
       params.form = form;
+
+      //Get the submission link
+      params.submissionLink = this.submissionLink(form);
```

Comment:
This seems wrong... to use the Form ID for the submission ID. How is this even working?...